XenAdmin/Wizards/GenericPages/SelectMultipleVMNetworkPage.cs

```diff
@@ -255,8 +255,11 @@ protected DataGridViewComboBoxCell FillGridComboBox(object xenRef)
 			return cb;
 		}
 
-        private bool ShowNetwork(Host targetHost, XenAPI.Network network)
+        private bool ShowNetwork(Host targetHost, XenAPI.Network network, string vsId)
         {
+            if (network.IsSriov())
+                return AllowSriovNetwork(network, vsId);

```

Comment:
Shouldn't this be: 
```if (network.IsSriov() && !AllowSriovNetwork(network, vsId))
    return false;;
```
so that we continue to do the other checks in this function even if the `AllowSriovNetwork` returns true?

Reply:
Oops,my bad, will correct it.